stacktracer/jvm/src/main/scala/scalaz/zio/internal/stacktracer/impl/AkkaTracer.scala

```diff
@@ -0,0 +1,32 @@
+package scalaz.zio.internal.stacktracer.impl
+
+import scalaz.zio.internal.stacktracer.ZTraceElement.{NoLocation, SourceLocation}
+import scalaz.zio.internal.stacktracer.{Tracer, ZTraceElement}
+
+import scala.util.matching.Regex
+
+final class AkkaTracer extends Tracer {
```

Comment:
Is it called `AkkaTracer` just to give credit (since it uses AkkaLineNumber under the hood)? My first reaction to this name was 'oh, does zio depend on akka now'. Maybe something like `AkkaInspiredTracer` would be a clearer name?

Reply:
I think there's little point in overt distancing, overly-precise terms, underscoring of differences or anything that could provoke more drama. This code isn't inspired, but straight up stolen 😉 and we're probably going to benefit from updates to the upstream version in the future, so I think it's ok to call it what it is.

Reply:
@kaishh I didn't have any of that in mind. I just shared that it was not clear to me that `AkkaTracer` is a standalone component and not an unexpected dependency on akka or akka-specific component. Anyway, I don't feel strongly about the names, but hoping you'll provide some context in the class doc.

Reply:
Fair enough! I've renamed it to `AkkaLineNumbersTracer` and qualified that it merely uses akka's bytecode parser, without incuring an akka dependency.